configuring/configuring-log-forwarding.adoc

```diff
@@ -116,6 +116,45 @@ Administrators can configure the following types of filters:
 include::modules/enabling-multi-line-exception-detection.adoc[leveloffset=+2]
 include::modules/logging-http-forward.adoc[leveloffset=+2]
 include::modules/cluster-logging-collector-log-forward-syslog.adoc[leveloffset=+2]
+
+
+[id="forwarding-logs-to-amazon-cloudwatch-from-sts-enabled-clusters_{context}"]
+== Forwarding logs to Amazon CloudWatch from STS enabled clusters
+
+Amazon CloudWatch is a service that helps administrators observe and monitor resources and applications on {aws-first}. You can forward logs from {loggingproductname} to CloudWatch securely by leveraging {aws-short}'s Identity and Access Management (IAM) Roles for Service Accounts (IRSA), which uses {aws-short} {sts-first}.
+
+The authentication with {aws-short} CloudWatch works as follows:
+
+* The log collector requests temporary {aws-short} credentials from {sts-fist} by presenting its service account token to the OpenID Connect (OIDC) provider in AWS. 
+* {aws-short} validates the token. Afterwards, depending on the trust policy, {aws-short} issues short-lived, temporary credentials (access key ID, secret access key, and session token) for the log collector to use.
+
+On {sts-short}-enabled clusters such as {product-rosa}, {aws-short) IAM roles are pre-configured with the required trust policies. This allows service accounts to assume the roles. Therefore, you can create a secret for {aws-short} with {sts-short} that uses the IAM role. You can then create or update a `ClusterLogForwarder` custom resource (CR) that uses the secret to forward logs to {aws-short} CloudWatch output. Follow these procedures to create a secret and a `ClusterLogForwarder` CR if roles have been pre-configured:
+
+* "Creating a secret for AWS CloudWatch with an existing AWS role"
```

Comment:
I'd prefer if these bullets are unquoted to more clearly demonstrate that they are steps to follow.    Also, should we make them links (bookmarks)?